pkg/expr/nodes.go

```diff
@@ -21,6 +21,8 @@ var (
 	logger = log.New("expr")
 )
 
+const FromExpressionHeaderName = "FromExpression"
```

Comment:
Since this is a header, we generally do `X-Grafana` , perhaps `X-Grafana-From-SSE`

Reply:
Oh, I'm not sure these become HTTP headers, and I guess it matches FromAlert, so okay to leave it.